<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackOrganiser</class>
 <widget class="QWidget" name="TrackOrganiser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>337</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutx">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="optionsBox">
       <property name="title">
        <string>Filenames</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="LineEdit" name="filenameScheme">
            <property name="minimumSize">
             <size>
              <width>288</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="configFilename">
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="BuddyLabel" name="label_6">
          <property name="text">
           <string>Filename scheme:</string>
          </property>
          <property name="buddy">
           <cstring>filenameScheme</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="vfatSafe">
          <property name="text">
           <string>VFAT safe</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="asciiOnly">
          <property name="text">
           <string>Use only ASCII characters</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="replaceSpaces">
          <property name="text">
           <string>Replace spaces with underscores</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="ignoreThe">
          <property name="text">
           <string>Append 'The' to artist names</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="files">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Original Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>New Name</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="UrlNoteLabel" name="ratingsNote">
     <property name="text" stdset="0">
      <string>Ratings will be lost if a file is renamed.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>support/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>BuddyLabel</class>
   <extends>QLabel</extends>
   <header>support/buddylabel.h</header>
  </customwidget>
  <customwidget>
   <class>UrlNoteLabel</class>
   <extends>QLabel</extends>
   <header>widgets/notelabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>vfatSafe</tabstop>
  <tabstop>asciiOnly</tabstop>
  <tabstop>replaceSpaces</tabstop>
  <tabstop>ignoreThe</tabstop>
  <tabstop>filenameScheme</tabstop>
  <tabstop>configFilename</tabstop>
  <tabstop>files</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
